src/main/java/io/phasetwo/keycloak/magic/resources/MagicLinkResource.java

```diff
@@ -71,6 +71,7 @@ public MagicLinkResponse createMagicLink(final MagicLinkRequest rep) {
             user,
             rep.getClientId(),
             rep.getRedirectUri(),
+            rep.getScopes(),
```

Comment:
Same as above. The request object needs a default (probably `openid`) so that it doesn't break backwards compatibility.

Reply:
Unfortunately, the previous value is null, not "openid" which is the primary source of the problems we have. Hence, the current implementation provides backward compatibility as far as I can say.